.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# See: https://help.github.com/articles/about-codeowners/
+
+# TODO: Add code owners
+# These owners will be the default owners for everything in the repo.
```

Comment:
Missing newline at the end of file. Please update all the similar files

Reply:
Done!